package/gluon-radv-filterd/src/respondd.c

```diff
@@ -0,0 +1,47 @@
+#include <respondd.h>
+
+#include <json-c/json.h>
+#include <libgluonutil.h>
+#include <net/ethernet.h>
+#include <stdio.h>
+
+#include "mac.h"
+
+static struct json_object * get_radv_filter() {
+	FILE *f = popen("exec ebtables -L RADV_FILTER", "r");
+	char *line = NULL;
+	size_t len = 0;
+	struct ether_addr mac = {};
+	struct json_object *ret = NULL;
+	char macstr[F_MAC_LEN + 1] = "";
+
+	if (!f)
+		return NULL;
+
+	while (getline(&line, &len, f) > 0) {
+		if (sscanf(line, "-s " F_MAC " -j ACCEPT\n", F_MAC_VAR_REF(mac)) == ETH_ALEN)
+			break;
+	}
+	free(line);
+
+	pclose(f);
+
+	snprintf(macstr, sizeof(macstr), F_MAC, F_MAC_VAR(mac));
```

Comment:
@lemoer noticed that would send 00:00:00:00:00:00 as gateway6_nexthop when no matching line is found. This is not the expected behavior.